### PR TITLE
[libsrt]Add new port.

### PR DESCRIPTION
**Pull request by JackBoosY**
This port does not support uwp.

Related: #8705.

### CODE REVIEW

File: ports/libsrt/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        -DENABLE_UNITTESTS=OFF
+        -DUSE_OPENSSL_PC=OFF
+)
+
+vcpkg_install_cmake()
+vcpkg_copy_pdbs()
+
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+
+
```
Comment (PhoebeHui) [inline ports/libsrt/portfile.cmake]: Remove the blank lines.

**Review by PhoebeHui — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks for the PR!

**Comment by maxlovic** [comment]
Thank you for this PR!
Do you have a badge showing SRT is included in vcpkg? We can put to the SRT repository.

**Comment by JackBoosY** [comment]
@maxlovic Well, we don't have badge now, or you can make a link to here?

**Comment by maxlovic** [comment]
@JackBoosY 
The badge is more to track the version of SRT in vcpkg. Like what we have for Debian.
image

And of course to highlight to the user that he can use vcpkg to build SRT.

I can create one manually, but this would not be very convenient.
So just asked maybe you had something.

**Pull request merged**